src/server/models/session.js

```diff
@@ -0,0 +1,22 @@
+module.exports = (sequelize, DataTypes) => {
+  const Session = sequelize.define('Session', {
+    name: DataTypes.STRING,
+    userId: DataTypes.INTEGER,
+    moduleId: DataTypes.INTEGER,
+    challengeId: DataTypes.INTEGER
+  });
+
+  Session.associate = (models) => {
+    Session.belongsTo(models.Challenge, {
+      foreignKey: 'challengeId',
+      onDelete: 'CASACADE'
+    });
+
+    Session.belongsToMany(models.User, {
```

Comment:
@mikey2020 
Between Session and User, there is a one to many relationship;
A user can have many sessions but a session can only belong to a user

Reply:
The current implementation is a **many-to-many** relationship